frontend/pages/[lng]/courses/[id]/manual-completions.tsx

```diff
@@ -96,7 +101,19 @@ const ManualCompletions = () => {
       )}
       <br />
       <Typography>
-        Format: csv with header with fields: user_id, grade
+        Completion date (optional) - if provided, will be default for every
+        completion with no date set
+      </Typography>
+      <LocalizationProvider dateAdapter={LuxonUtils}>
+        <DatePicker
+          inputFormat="yyyy-MM-dd"
+          placeholder="yyyy-MM-dd"
+          onChange={setCompletionDate}
+          value={completionDate}
+        />
+      </LocalizationProvider>
+      <Typography>
+        Format: csv with header with fields: user_id,grade[,completion_date]
```

Comment:
Format: csv with header with fields: user_id,grade[,completion_date in ISO 8601 format]